frontend/src/utils.js

```diff
@@ -1,6 +1,6 @@
 function arrayToHex(iv) {
   let hexStr = '';
-  for (let i in iv) {
+  for (let i in iv) { // eslint-disable-line prefer-const
```

Comment:
But possibly same  here... Try running `npm run format` to let **prettier** reformat everything, and I'd guess that it'll move the comment to the following line and you'd get ESLint errors if you ran `npm run lint`.

👍 to using the same `// eslint-disable-next-line prefer-const` as above, and moving it up to line 2.5 above.